agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/configuration/Configuration.java

```diff
@@ -1398,14 +1398,18 @@ public static class ProfilerConfiguration {
     public int configPollPeriodSeconds = 60;
     public int periodicRecordingDurationSeconds = 120;
     public int periodicRecordingIntervalSeconds = 60 * 60;
-    public boolean enabled = false;
+    @Nullable public Boolean enabled;
     public String memoryTriggeredSettings = "profile-without-env-data";
     public String cpuTriggeredSettings = "profile-without-env-data";
     public String manualTriggeredSettings = "profile-without-env-data";
     @Nullable public String serviceProfilerFrontEndPoint = null;
     public boolean enableDiagnostics = false;
     public boolean enableRequestTriggering = false;
     public RequestTrigger[] requestTriggerEndpoints = {};
+
+    public boolean isEnabled() {
+      return !Boolean.FALSE.equals(enabled);
```

Comment:
double negative, could simplify?

Reply:
this deals with the fact that enabled could be null, alternative is:

```
 enabled == null || enabled
```